packages/grafana-ui/src/components/Modal/Modal.story.tsx

```diff
@@ -53,8 +53,8 @@ export const Basic: Story = ({ body, title, ...args }) => {
     <Modal title={title} {...args}>
       {body}
       <Modal.ButtonRow>
-        <Button>Button1</Button>
         <Button variant="secondary">Cancel</Button>
```

Comment:
cancel button should use fill=outline style

Reply:
![Screenshot from 2023-02-10 09-44-02](https://user-images.githubusercontent.com/10999/218044983-8aed0621-b2b6-4d28-9408-f78d628861a7.png)
![Screenshot from 2023-02-10 09-43-48](https://user-images.githubusercontent.com/10999/218044986-b7263af7-f9aa-4c6d-945e-27717ccbc34c.png)
![Screenshot from 2023-02-10 09-42-47](https://user-images.githubusercontent.com/10999/218044988-a3129f7f-18fc-487f-b071-ac4ffe41f21b.png)
![Screenshot from 2023-02-10 09-42-35](https://user-images.githubusercontent.com/10999/218044990-b5920e5a-4df2-47da-9bf7-3681bfd8afba.png)